bayesian/api/user_api.py

```diff
@@ -0,0 +1,58 @@
+"""Definition of all REST API endpoints for user management."""
+import uuid
+
+from flask import Blueprint, request
+from flask.json import jsonify
+
+from fabric8a_auth.auth import login_required
+from fabric8a_auth.errors import AuthError
+
+from bayesian.exceptions import HTTPError
+from bayesian.utility import user_utils
+from bayesian.utility.user_utils import UserException
+
+user_api = Blueprint('user_api', __name__, url_prefix='/user')
+
+
+@user_api.route('/<user_id>', methods=['GET'])
+@login_required
+def get_user(user_id):
+    """Endpoint for getting user details."""
+    if not user_id:
+        raise HTTPError(400, "user id should be present")
+
+    user = user_utils.get_user(user_id)
+    return jsonify(user_id=user.user_id, snyk_api_token=user.snyk_api_token)
```

Comment:
What is the purpose of returning `snyk_api_token` through API? Is it just to check whether the UUID is associated with registered snyk account? If so, can abstract it with `registration_status` instead of `snyk_api_token`?

Reply:
Yes this can be changed to status